test/functional/tests/cli/test_cli_start_stop.py

```diff
@@ -80,3 +82,32 @@ def test_cli_add_remove_default_value(shortcut):
         TestRun.fail("No cache should be present after stopping the cache")
     if output.stdout != "No caches running":
         TestRun.fail(f"Invalid message, expected 'No caches running', got {output.stdout}")
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+def test_cli_load_and_force():
+    """
+        title: Test if it is possible to use 'load' and 'force' flag at once
+        description: |
+          Start new cache instance
+          Stop cache instance
+          Try to start cache with 'load' and 'force' options at the same time
+        pass_criteria:
+          - Command should fail
```

Comment:
Maybe "Start cache command with both 'force' and 'load' options should fail" ? Because it seems like any command should fail